packages/ckeditor5-dom-converter/src/ConversionApi.ts

```diff
@@ -47,8 +63,23 @@ export class ConversionApi {
    * @param qualifiedName - qualified name of element to create
    */
   createElementNS(namespaceURI: string | null, qualifiedName: string): Element {
-    const { targetDocument } = this;
-    return targetDocument.createElementNS(namespaceURI, qualifiedName);
+    return this.targetDocument.createElementNS(namespaceURI, qualifiedName);
+  }
+
+  /**
+   * Creates a `Range` owned by `targetDocument`.
```

Comment:
Well, actually, it is an empty range. We would have to dive into the DOM API even more, as this may be identical with `new Range()`. I don't even think, a range has any binding to the document itself. But it may be used as API to create it.

Thus, no change here. We may investigate more, to understand the API better and describe it better here.